h2o-probes.d

```diff
@@ -22,6 +22,17 @@
 
 struct st_h2o_tunnel_t;
 
+/* @appdata
+{
+    "receive_request_header": ["value"],
+    "send_response_header": ["value"],
+    "h3_packet_receive": ["bytes"],
```

Comment:
I do not think we need this, as it is the bytes sent over the wire.

Reply:
Agree with @kazuho.